### PR TITLE
Updating so that it works with django's updated command API

### PR DESCRIPTION
**Pull request by vikki**
since 1.8 django argument parsing has changed
Also API for requesting models has changed (old way is deprecated)

### CODE REVIEW

File: django_yadt/management/commands/yadt_refresh_variant.py [hunk 1]
```diff
@@ -4,16 +4,21 @@
 
 from ...utils import get_variant
 
+
 class Command(BaseCommand):
-    USAGE = "<app_label> <model> <field> <variant>"
+    def add_arguments(self, parser):
+        parser.add_argument('app_label', type=str)
```
Comment (lamby) [inline django_yadt/management/commands/yadt_refresh_variant.py]: is `type=str` not the default?

**Review by lamby — CHANGES_REQUESTED** [review]

File: django_yadt/management/commands/yadt_refresh_variant.py (hunk 1 above)
Comment (vikki) [reply django_yadt/management/commands/yadt_refresh_variant.py]: Yes, looks like it. Normally I'd prefer that it was more explicit, but I guess it's just noise here so I've taken it out

File: django_yadt/management/commands/yadt_refresh_variant.py [hunk 2]
```diff
@@ -4,16 +4,21 @@
...
-            app_label, model_name, field_name, variant_name = args
-        except ValueError:
-            raise CommandError(self.USAGE)
-
-        variant = get_variant(app_label, model_name, field_name, variant_name)
+        variant = get_variant(
+            options['app_label'],
+            options['model_name'],
+            options['field_name'],
+            options['variant_name']
```
Comment (lamby) [inline django_yadt/management/commands/yadt_refresh_variant.py]: missing tailing comma

**Comment by lamby** [comment]
:)

**Review by tomokas — APPROVED** [review]
:+1:

**Pull request merged**